### PR TITLE
Add setup readme

### PR DESCRIPTION
**Pull request by KatieLG**

### CODE REVIEW

File: docs/assets/collab-join.png
Comment (JustCallMeRay) [inline docs/assets/collab-join.png:1]: Can we get join selected over create new

**Review by JustCallMeRay — CHANGES_REQUESTED** [review]

**Comment by JustCallMeRay** [comment]
I'll merge it in on Tuesday if you don't have time

File: docs/assets/collab-join.png
Comment (KatieLG) [reply docs/assets/collab-join.png:1]: Yeah good shout I'll fix that quickly when I'm home

Comment (KatieLG) [reply docs/assets/collab-join.png:1]: done

**Pull request merged**